src/components/BreadCrumbs.tsx

```diff
@@ -2,15 +2,37 @@ import React from "react"
 import { AlbumListItem } from "../data/types"
 import { Link } from "react-router-dom"
 import homeIcon from "../assets/icon-color.png"
+import { BaseProps } from "../data/types"
 
-type BreadcrumbsProps = {
+interface BreadcrumbsProps extends BaseProps {
   breadcrumbs: AlbumListItem[]
   bg?: boolean
 }
 
-function BreadCrumbs({ breadcrumbs, bg }: BreadcrumbsProps) {
-  if (!breadcrumbs.length) return null
-
+function BreadCrumbs({ breadcrumbs, bg, isLoading }: BreadcrumbsProps) {
+  if (isLoading) {
+    return (
+      <div className={`main-crumb-wrapper ${bg && "with-bg"}`}>
+        <div className="crumb-nav-wrap">
+          <nav className="nav">
+            <div className="breadcrumb-wrap">
+              <Link to="/">
+                {" "}
+                <img src={homeIcon} alt="home-icon" width={25} height={25} />
+              </Link>
+            </div>
+            <div className="breadcrumb-wrap">
```

Comment:
Would it be possible to check for `isLoading` at this point, instead of at the top level, given that the home icon will always be present?

```tsx
const BreadCrumbs = ({ breadcrumbs, bg, isLoading }: BreadcrumbsProps) =>
  <div className={`main-crumb-wrapper ${bg && "with-bg"}`}>
    <div className="crumb-nav-wrap">
      <nav className="nav">
        <div className="breadcrumb-wrap">
          <Link to="/">
            {" "}
            <img src={homeIcon} alt="home-icon" width={25} height={25} />
          </Link>
        </div>
        { isLoading
          ? // loading JSX
          : // real breadcrumbs }
      </nav>
    </div>
  </div>
```

Reply:
Yes, thanks for pointing it out, that's even more readable :)